StepDefinitions/AddlangStepDefinitions.cs

```diff
@@ -0,0 +1,124 @@
+using OpenQA.Selenium;
+using System;
+using SpecProj2.Pages;
+using TechTalk.SpecFlow;
+using System.Collections.Generic;
+using TechTalk.SpecFlow;
+using OpenQA.Selenium.Chrome;
+using NUnit.Framework;
+using System.Reflection.Emit;
+using OpenQA.Selenium.Support.UI;
+
+namespace SpecProj2.StepDefinitions
+{
+    [Binding]
+    public class AddlangStepDefinitions
+    {
+        IWebDriver driver = new ChromeDriver();
+        Login Loginobj = new Login();
+        Addmultiplelanguage Addmultiplelanguageobj = new Addmultiplelanguage();
+        Editlanguage Editlanguageobj = new Editlanguage();
+        Deletelanguage deletelanguageobj = new Deletelanguage();
+
+        [Given(@"User logs into Mars Portal")]
+        public void GivenUserLogsIntoMarsPortal()
+        {
+            Loginobj.login(driver);
+        }
+        [When(@"User creates a new Language record '([^']*)'")]
+        public void WhenUserCreatesANewLanguageRecord(string language)
+        {
+            Addmultiplelanguageobj.cleardata(driver);
+            Addmultiplelanguageobj.addlanguage(driver, language);
+        }
+        [Then(@"new Language '([^']*)' should be successfully created")]
+        public void ThenNewLanguageShouldBeSuccessfullyCreated(string language)
+        {
+            Addmultiplelanguageobj.verifyaddedinlist(driver, language);
```

Comment:
Please do not pass the driver In action methods (methods which in pages doing things like create, delete etc ) instead get the pages to extend to common driver where you will create a common driver class which will initialise the web driver 

public class CommonDriver
    {
        public static IWebDriver driver;

        public void initialise()
        {
            // create driver
            driver = new ChromeDriver();

        }

    }

Reply:
fix this everywhere you have done this